### DISCUSSION TITLE
[#991909] UI size resets when switching dimensions

### DISCUSSION THREAD

**Discussion by MisledWater79** [Mojang/minecraft-editor#11] [Bugs]
I went to the nether using the world options tab and it reset the UI scale from what I had previously.

**Comment by minecraftchloe** [comment]
Thank you! Repro'd!

**Reply by minecraftchloe** [reply]
Duplicate to [UI resets when changing dimensions · Mojang/minecraft-editor · Discussion #33 (github.com)](https://github.com/Mojang/minecraft-editor/discussions/33)

**Comment by minecraftchloe** [comment]
Should be fixed! Let us know if this is an issue again!
